post_install.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+arduino_esp32_rules () {
+    echo ""
+    echo "# Arduino ESP32 bootloader mode udev rules"
+    echo ""
```

Comment:
This is a particularly weird mixture of echo and "here" documents. Simply use one "here" document.